### ISSUE TITLE
Interfaces Created as Virtual for Physical Hosts

### ISSUE THREAD

**Issue by abhilashchinthalapudi** [synackray/vcenter-netbox-sync#66]
Thanks for the great tool btw.This is so cool

I have observed that the Script creates Virtual Interfaces for the Physical ESXi hosts as well which cannot be cabled and connected to switches.I am not sure if this is intentional but want to let here know

I think this is the code that does that 
results["interfaces"].append(
                            {
                                "device": {"name": obj_name},
                                # Interface speed is placed in the description
                                # as it is irrelevant to making connections and
                                # an error prone mapping process
                                "type": 32767, # 32767 = Other

I think the above can be fixed by changing the type to 1150.

Apart from that everything looks perfect.

**Comment by synackray** [comment]
Hello @abhilashchinthalapudi! Using the VMware SDK we collect both the physical and virtual interfaces for the vCenter hosts. While the vCenter virtual interfaces cannot be physically connected there should not be any issue connecting the physical interfaces to other equipment.

In the image below the vmk0 - 2 interfaces are configured in vswitches using vmnic0 and vmnic1 physical interfaces as uplinks.
![image](https://user-images.githubusercontent.com/24757919/74358015-05d39280-4dc1-11ea-864b-433cb6eb9ce0.png)

In the second screenshot I have attached the physical NICs themselves to the top of rack switches.
![image](https://user-images.githubusercontent.com/24757919/74358418-af1a8880-4dc1-11ea-917e-a17182ace3fe.png)


Is this the behavior you desire or am I perhaps misunderstanding? Is there a particular need to attach the virtual interfaces themselves?

**Comment by abhilashchinthalapudi** [comment]
@synackray I think I might have modified something on the code on my end .I tested the current develop code and it works perfectly.Sorry for the confusion and thanks for the Response.
I will wait for the master release before putting it into production.
Thanks for the Primary IP as well that works perfectly as well

Just a quick Question .When I run the Script on the first time it does not create the ip addresses but if I run it again it creates the IP addresses. Am I missing anything that needs to be configured in settings.py

I encountered another issue with 
url = req.json()["next"]
It works fine for the first loop of Objects but for the next loop it fails because we disabled http and only https is allowed and I was not sure how the first hop worked but replacing it with below works fine 
              urlnext = req.json()["next"]
              url = urlnext.replace("http","https")
Is it some bad config on my end ?
